File: abc/abc219/a.cs
// https://atcoder.jp/contests/abc219/submissions/46503021

using System;

public class A
{
  static void Main()
  {
    int x = int.Parse(Console.ReadLine());
    if(x < 40)
    {
      Console.WriteLine(40 - x);
    }
    else if(x < 70)
    {
      Console.WriteLine(70 - x);
    }
    else if(x < 90)
    {
      Console.WriteLine(90 - x);
    }
    else
    {
      Console.WriteLine("expert");
    }
  }
}

File: abc/abc219/a.rs
// https://atcoder.jp/contests/abc219/submissions/47564504

use proconio::input;

fn main() {
    input! {
        x: u8
    }
    
    if x < 40 {
        print!("{}", 40 - x);
    } else if x < 70 {
        print!("{}", 70 - x);
    } else if x < 90 {
        print!("{}", 90 - x);
    } else {
        print!("expert");
    }
}

